core/modules/simpletest/tests/menu.test

```diff
@@ -49,7 +49,7 @@ class MenuWebTestCase extends BackdropWebTestCase {
     $pass = ($pass && empty($parts));
 
     $this->assertTrue($pass, format_string('Breadcrumb %parts found on @path.', array(
-      '%parts' => implode(' » ', $trail),
+      '%parts' => implode(' ', $trail),
```

Comment:
Feeble attempt to fix failing test. I'm not able to `dsm` (or the like) to see what this function is getting from the page, or what it's building from the data being passed to it for testing against the page. Not sure how to debug this :cry: